README.md

```diff
@@ -23,6 +23,17 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
 ### Create reCAPTCHA account
 [Create ReCAPTCHA account](http://www.google.com/recaptcha/admin)
 
+1. Sign up at [ReCAPTCHA](http://www.google.com/recaptcha/admin) using the following configurations:
+
+|Field         |Value        |
+|--------------|-------------|
+|Label         |`short`      |
+|reCAPTCHA type|`reCAPTCHAv3`|
+|Domains       |`localhost`  |
+
+Copy `SECRET KEY` into `.env` file and rename the variable to `RECAPTCHA_SECRET`.
+Copy 'SITE_KEY' into `app/adapter/web/.env.development` file and rename the variable to `REACT_APP_ RECAPTCHA_SITE_KEY` for use when building the docker image.
```

Comment:
```suggestion
Copy `SITE_KEY` into `app/adapter/web/.env.development` file and rename the variable to `REACT_APP_RECAPTCHA_SITE_KEY`.
```

Reply:
`REACT_APP_RECAPTCHA_SITE_KEY` is used by `create-react-app` to fill the environmental variables into the frontend.

Reply:
Fixed.